node-commons/src/main/java/org/apache/iotdb/commons/utils/PathUtils.java

```diff
@@ -202,4 +203,16 @@ private static boolean checkBackQuotes(String src) {
     }
     return src.length() == (src.replace("``", "").length() + num);
   }
+
+  /** Add a prefix to a relative path file */
+  public static String addPrefix2FilePath(String prefix, String file) {
```

Comment:
Perhaps we should put this method in FileUtils. PathUtils is for PartialPath.

Reply:
fixed